plugins/s3/src/index.mjs

```diff
@@ -1106,6 +1186,265 @@ const GetObject = {
   },
 }
 
+const GetObjectAcl = {
+  awsDoc: docRoot + 'API_GetObjectAcl.html',
+  validate: {
+    Bucket,
+    Key,
+    VersionId,
+    ...getValidateHeaders('ExpectedBucketOwner', 'RequestPayer'),
+  },
+  request: (params, utils) => {
+    const { Key } = params
+    const queryParams = [ 'VersionId' ]
+    const headers = getHeadersFromParams(params, queryParams)
+    const query = { 'acl': '', ...getQueryFromParams(params, queryParams) }
+    const { host, pathPrefix } = getHost(params, utils)
+    return {
+      host,
+      pathPrefix,
+      path: `/${Key}`,
+      headers,
+      query,
+    }
+  },
+  response: ({ payload, headers }) => {
+    const { AccessControlList, Owner } = payload
+    let { Grant: Grants } = AccessControlList
+    if (!Array.isArray(Grants)) {
+      Grants = [ Grants ]
+    }
+    Grants = Grants.map(i => {
+      let result = { ...i }
+      let { Grantee } = result
+      Grantee.Type = Grantee['xsi:type']
+      delete Grantee['xsi:type']
+      delete Grantee['xmlns:xsi']
+      result.Grantee = Grantee
+      return result
+    })
+    let { RequestCharged } = parseHeadersToResults({ headers })
+    let result = {
+      Owner,
+      Grants,
+    }
+    if (RequestCharged) result.RequestCharged = RequestCharged
+    return result
+  },
+}
+
+// TODO: allow paginator tokens in headers
```

Comment:
Try this out with and lmk how it goes! https://github.com/architect/aws-lite/pull/150